### ISSUE TITLE
PrdouctSet module enabled?

### ISSUE THREAD

**Issue by cyfranT** [allegro/allegro-api#9824]
Przy aktualizacji ofert z zestawami otrzymalismy ostatnio takie błędy zwrotnie:
`{"code":"OfferRequiresProductSetModule","message":null,"details":null,"path":null,"userMessage":"The offer needs to have a product set module enabled.","metadata":{}}`

Co to za błąd i gdzie się aktywuje ów **product set module** ?

**Comment by PrzemyslawLukanowski** [comment]
Wygląda to na problem po naszej stronie, zweryfikujemy to i wrócę z informacją. Ponowienie requestu powinno pomóc w tej sytuacji.

**Comment by PrzemyslawLukanowski** [comment]
Komunikat był spowodowany chwilowymi problemami po naszej stronie. Rozumiem, że po powtórzeniu requestu problem już nie występuje?

**Comment by cyfranT** [comment]
Tak, po 5 września już nie było tego problemu. Dzięki za weryfikację.
